src/GraphQL/Instrumentation/InstrumentFieldsMiddleware.cs

```diff
@@ -11,15 +11,17 @@ public class InstrumentFieldsMiddleware : IFieldMiddleware
         /// <inheritdoc/>
         public async Task<object?> Resolve(IResolveFieldContext context, FieldMiddlewareDelegate next)
         {
+            var name = context.FieldAst.Name.StringValue; //ISSUE:allocation
```

Comment:
Not an issue.  Boxed ROM is worse than a string.

Reply:
Just a reminder.